### DISCUSSION TITLE
Markdown table not rendered in frontend

### DISCUSSION THREAD

**Discussion by Eiscoffee** [requarks/wiki#4962] [Error / Bug Report]
Hello,

a typical Markdown-table looks like this:

```
|H1|H2|H3|
|---|---|---|
|C1|C2|C3|
```

Now i want to have a Table without header.
I red, that following git was implemented:
https://github.com/RedBug312/markdown-it-multimd-table

so i think this should work:

```
|--|--|--|
|C1|C2|C3|
```

in the backend (the markdown editor for my page) it works and is rendered correctly on the right hand side.
If i save it and look to the live page, it does not work.

Can someone help?

edit:

even this works in the backend:

```
|
C1|C2|C3
```

**Comment by markus-96** [comment]
you only have to enable "MultiMarkdown Table" in the Adminpage, under "Rendering" -> "markdown".
